tests/core/test_actions.py

```diff
@@ -70,7 +70,7 @@ def template_nlg():
             }
         ],
     }
-    return TemplatedNaturalLanguageGenerator(templates)
+    return TemplatedNaturalLanguageGenerator(responses)
 
 
 @pytest.fixture(scope="module")
```

Comment:
What does the `template` refer to here in `template_sender_tracker`?

Reply:
I think nothing. I think it only includes `template` because it's used to test the template generator. However, there is nothing special about this tracker 🤯

Reply:
I'll just leave it then. 😅